compass/landice/landice.py

```diff
@@ -0,0 +1,71 @@
+import numpy as np
+
+
+def flood_fill(thk, vx, vy):
+    """
+    Remove glaciers and ice-fields that are not connected to the ice sheet.
+
+    Parameters
+    ----------
+    thk : numpy.ndarray
+        Ice thickness from gridded dataset
+    vx : numpy.ndarry
+        Velocity x-component from gridded dataset
+    vy : numpy.ndarray
+        Velocity y-component from gridded dataset
```

Comment:
Could you explain in the docstring why velocity components are needed for the flood fill?  It might be obvious to you but it was not to me.  If the idea is to trip 3 arrays, I would generalize this to just take thickness and a list of arrays (or better yet an `xarray.Dataset` to trim.  That way, you don't have to generalize if you need other trimmed variables in the future.

Reply:
Thanks for the suggestion. That's a much better approach.

Reply:
I could also have the only input be thickness, and then have it return floodMask. Then the actual trimming would be performed after calling flood_fill(). That might be more flexible and a bit easier to understand.

Reply:
I like that better!

Reply:
Fixed in commit [52d2c97](https://github.com/MPAS-Dev/compass/pull/315/commits/52d2c97c6362d6bc8a4cadd647130174ce44fbb5)